### ISSUE TITLE
Load my opened library automatic or extend 'Refresh' to any type's context menu item.

### ISSUE THREAD

**Issue by 2300070** [icsharpcode/ILSpy#3207]
**Is your feature request related to a problem? Please describe.**
It's not a bug but it may enhance user experience when it is resolved.

**Describe the solution you'd like**
I'd like the application load modified assembly automatic, rather than I right click 'Reload' menu. it is prefect if the explorer remembers the  type or member that selected.

**Additional context**
no other info.

**Comment by christophwille** [comment]
No, that feature creates more UI problems (think of multiple open assemblies part of a build, all modified at once - or an assemblies being deleted); ILSpy uses files always read-only, so there won't be (future) write conflicts, so having latest or being notified about latest is not central concern.
